### ISSUE TITLE
double posts

### ISSUE THREAD

**Issue by l0g4n1337** [ayushunofficial/ig-to-discord#2]
The bot often posts duplicate posts.
The strange thing is that they are not posted at the same time, but post a copy after X minutes.

![Discord_mAYANnPKAW](https://user-images.githubusercontent.com/49865595/167087881-121690d0-69ed-4d73-bfba-776c59a06dce.png)

![Discord_8gk76RXdDB](https://user-images.githubusercontent.com/49865595/167087893-8c99af07-d2e9-4671-9103-5f6a89646d48.png)

**Comment by ayushh2k** [comment]
did you check for duplicates in users.json? and where do you host the bot?

**Comment by l0g4n1337** [comment]
Yes, I checked for duplicates in users.json, and there aren't any.
I host the bot on replit.
I know that it is not a stable service, and it happens that sometimes bots go off and then come back on after a short time.
Could it be due to this?

**Comment by ayushh2k** [comment]
I couldnt replicate the issue on replit but it might be because it goes off before the latest post's shortcode gets updated in users.json, try running it on your machine or use a different hosting provider.

**Comment by ayushh2k** [comment]
also i've figured that json files are not the best way to store the shortcodes as sometimes it can get corrupted so I will try to implement a database for the post codes.
